recipes/dcmtk/all/patches/0001-cmake-use-conan-packages.patch

```diff
@@ -12,8 +12,14 @@
 +      set(LIBXML_LIBS ${CONAN_LIBS_LIBXML2})
        message(STATUS "Info: DCMTK XML support will be enabled")
        set(WITH_LIBXML 1)
++      if(NOT WITH_LIBXML_SHARED)
        # this hides some warnings that are emitted when linking against libxmlXXX.lib instead of linking the DLL directly
-@@ -58,8 +58,8 @@
+       add_definitions("-DLIBXML_STATIC")
++      endif()
+     else() # turn off library if library path not set
+       message(STATUS "Warning: XML support will be disabled because libxml2 directory is not specified. Correct path and re-enable DCMTK_WITH_XML.")
+       set(DCMTK_WITH_XML OFF CACHE BOOL "" FORCE)
+@@ -58,8 +60,8 @@ if(WIN32 AND NOT MINGW)
```

Comment:
When creating a patch, I always try to keep the line offsets in the hunks the same (e.g. `@@ -40,4 +40,4 @@` and not make it `@@ -40,4 +42,4 @@`.
It would be nice if you could keep it that way.
You can do this by removing some unneeded lines such as `message` lines.

This is just a personal preference, but makes modifying patches manually easier.

Reply:
CMake doesn't allow multiple commands on one line.
What I do is remove comments and empty lines.